src/options.html

```diff
@@ -21,6 +21,14 @@
 			<input class="form__input" type="text" id="customNewTabUrl" placeholder="https://example.com">
 		</div>
 
+		<div class="form__field">
+			<label class="form__label">New Tab Title</label>
+
+			<span class="form__hint">The text that is shown in the new browser tab.</span>
+
```

Comment:
Can you re-word this to be `The text that is shown in the tab bar for the new tab`